src/build.d

```diff
@@ -488,6 +500,46 @@ alias style = makeDep!((builder, dep)
         ]);
 });
 
+alias man = makeDep!((builder, dep) {
+    alias genMan = methodInit!(Dependency, (genManBuilder, genManDep) => genManBuilder
+        .target(env["G"].buildPath("gen_man"))
+        .sources([
+            dmdRepo.buildPath("docs/gen_man.d"),
+            env["D"].buildPath("cli.d")])
+        .command([
+            env["HOST_DMD_RUN"],
+            "-I" ~ srcDir,
+            "-of" ~ genManDep.target]
+            ~ flags["DFLAGS"]
+            ~ genManDep.sources)
+        .msg(genManDep.command.join(" "))
+    );
+
+    const genManDir = env["GENERATED"].buildPath("docs/man");
+    alias dmdMan = methodInit!(Dependency, (dmdManBuilder, dmdManDep) => dmdManBuilder
+        .target(genManDir.buildPath("man1/dmd.1"))
+        .deps([genMan, mkdirDep(dmdManDep.target.dirName)])
+        .commandFunction(() {
+            if (updateIfChanged(dmdManDep.target, genMan.target.execute.output))
```

Comment:
Why not always write the output file?
If `cli.d` is newer than the output file, the build script will now always try to regenerate the output if it didn't change as the timestamp of `cli.d` would be newer.